fedbiomed/node/node.py

```diff
@@ -55,10 +56,11 @@ def __init__(self,
             researchers=[ResearcherCredentials(port=res['port'], host=res['ip'], certificate=res['certificate'])],
             on_message=self.on_message,
         )
-        # Note: `PendingRequests` `NodeToNodeRouter` should not be changed to singleton.
+        # Note: `EventWaitExchange` `NodeToNodeRouter` should not be changed to singleton.
```

Comment:
Hi @mvesin, 
I think we can remove this comment since we do not plan to code them as singleton.

Reply:
@srcansiz Ok I remove it